### ISSUE TITLE
Frog light editor colour scheme lacks contrast

### ISSUE THREAD

**Issue by marijnvdwerf** [decompme/decomp.me#758]
Some of the colours in the default light theme don't contrast enough with the background. The green/yellow backgrounds have a contrast ratio of about 2, which definitely isn't enough for small text. (WCAG recommends like 4.5, but I think that would affect the "fresh" feel of the colour scheme too much)

<img width="730" alt="image" src="https://github.com/decompme/decomp.me/assets/737603/e20b7e57-20ab-4f87-9708-56b8b0334629">

<img width="506" alt="image" src="https://github.com/decompme/decomp.me/assets/737603/8bf7ac78-a46e-4eb0-b975-c84a4d52ae89">
